### PR TITLE
Show clickable names of hidden files

### LINKED ISSUE #57: Show names/count of hidden files
> Show 14 hidden items: .github, .babelignore, .babelrc.js, .editorconfig, .eslintignore, .eslintrc, ...

or

> Hidden: .github, .babelignore, .babelrc.js, .editorconfig, .eslintignore, .eslintrc, and 8 more

Or even better, make each item clickable:

screen shot 2017-07-11 at 10 34 43

`Hidden:` and `8 more` would still open the entire list on click.

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #57 

Based on #59 (ignore DRY commit), only look at https://github.com/sindresorhus/hide-files-on-github/pull/60/commits/11f09252038302c4a08462a0832e4f3d3aacb436 for the links and https://github.com/sindresorhus/hide-files-on-github/pull/60/commits/76d2c7b88d8295e1643e77328e8b67231cf76a8b for the chevron.

demo

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -25,6 +30,12 @@
 .hide-files-toggle:checked ~ .files .hide-files-btn:before {
 	content: 'Show ';
 }
+.hide-files-toggle:not(:checked) ~ .files .hide-files-row .hide-files-list {
+	display: none;
+}
+.hide-files-list a {
+	margin-left: 1em;
```
Comment (bfred-it) [inline extension/content.css]: Distance is detabable

**Comment by bfred-it** [comment]
Possible addition: use GitHub's own collapsible `>` chevron instead of the **Show/hide collapsibles** text 
screen shot 2017-07-12 at 00 08 51

**Comment by bfred-it** [comment]
If you can deal with the non-root listing order being:

screen shot 2017-07-12 at 00 14 21

instead of #59's

screen shot 2017-07-12 at 00 14 04

Then this PR can be CSS only 👍

**Comment by radiovisual** [comment]
So I may have given this feature a hard time at first, but now I like it. :rofl: 

Does it interfere/override or work in tandem with the RegExp in the options?

**Comment by bfred-it** [comment]
👍 works in tandem, it shows the links it hides.

**Comment by sindresorhus** [comment]
I have to admit this is a really good POC ✨

This feature is still not something I would use, but I'm ok with adding it as an option.

**Comment by sindresorhus** [comment]
In https://github.com/babel/babel, the `...` are too close to the edge:

screen shot 2017-07-15 at 19 49 11

**Comment by bfred-it** [comment]
Also something to tackle when I have time to fix this: sub-folder links like

screen shot 2017-07-18 at 02 00 10


https://github.com/mozilla/web-ext

**Comment by bfred-it** [comment]
Fixed both issues 👍

**Comment by bfred-it** [comment]
I felt that f50f5fe was needed to improve wrapping of long lists (i.e. avoid ellipsis mid-link)

Here's a couple of before and after

### babel

before

after

### web-ext

before

after

**Comment by sindresorhus** [comment]
The files are showing as blue links for me now, which is IMHO too noisy.

**Comment by sindresorhus** [comment]
> This feature is still not something I would use, but I'm ok with **adding it as an option.**

@bfred-it What are your thoughts on? :arrow_up:

**Comment by bfred-it** [comment]
Blue links were a mistake, will fix tomorrow.

I can add it as an option, but what do you think about making it enabled by default? This way new users can find it useful and old users will be able to revert it.

Also: what's the toggle button's label if they disable this option? Chevron or text?

**Comment by sindresorhus** [comment]
1. Sure.
2. Text, like it is now, I would say.

**Comment by bfred-it** [comment]
`Make it an option` looks like
 
screen shot 2017-07-20 at 16 37 16

**Comment by bfred-it** [comment]
`Add visual example to options` looks like

screen shot 2017-07-21 at 14 41 45

**Comment by bfred-it** [comment]
Both style work so feel free to pick/discard and merge if you like any.

**Comment by sindresorhus** [comment]
Really nice touch on the settings look ❤️

**Comment by sindresorhus** [comment]
Can you update the text in the readme "In the options, you can choose what files/directories to hide with a custom regex." with something that describes the two styles.

**Comment by bfred-it** [comment]
screenshot of the readme

**Comment by sindresorhus** [comment]
Amazing stuff ✨

**Pull request merged**